easybuild/easyconfigs/a/AmberTools/AmberTools-20-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'EB_Amber'
+
+name = 'AmberTools'
+version = '20'
+versionsuffix = '-Python-%(pyver)s'
+
+
+homepage = 'https://ambermd.org/'
+description = """AmberTools consists of several independently developed packages that work well by themselves,
+ and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations,
+ with either explicit water or generalized Born solvent models."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'usempi': True}
+
+# download requires registration, see http://ambermd.org/AmberTools17-get.html
+local_download_credentials = '?Name=Easybuild&Institution=Easybuild&City=Internet&State=Other&Country=Belgium'
+source_urls = ['https://ambermd.org/cgi-bin/AmberTools20-get.pl']
+sources = [{
+    'download_filename': local_download_credentials,
+    'filename': SOURCE_TAR_BZ2,
+}]
+patches = ['Amber-16_fix-hardcoding.patch']
+checksums = [
+    'b1e1f8f277c54e88abc9f590e788bbb2f7a49bcff5e8d8a6eacfaf332a4890f9',  # AmberTools-20.tar.bz2
+    'a71f1bd70bac8d5e25386d6db90e95874ad300b956ed5b5afbc8848b7c1e00dc',  # Amber-16_fix-hardcoding.patch
```

Comment:
@deniskristak the checksum is not correct. did you change anything to this file?

Reply:
yes, I did, I needed to add the patch file as well. It should now be fine